Running provides the following error: 

./PizzaOrder.java:34: error: no suitable constructor found for Pizza(String,int,int,int)
    m_order[0] = new Pizza("small", 1, 0, 0);
                 ^
    constructor Pizza.Pizza() is not applicable
      (actual and formal argument lists differ in length)
    constructor Pizza.Pizza(String,int,int,int,int) is not applicable
      (actual and formal argument lists differ in length)
    constructor Pizza.Pizza(Pizza) is not applicable
      (actual and formal argument lists differ in length)
1 error

I believe adding an argument for the new garlic variable to the PizzaOrder.java constructor will fix this.

--- a/PizzaOrder.java
+++ b/PizzaOrder.java
@@ -7,8 +7,9 @@
  * The purpose of this class is to create a PizzaOrder which is an array of type Pizza
  * It also calculates the total cost of the order, utilizing the calcCost() method from the Pizza class
  * Note: I added the instance variable pizzasCurrentlyInOrder so the addPizza() method knows when the order is already full.
- * @version 1.0
+ * @version 1.1
  */
+
 public class PizzaOrder {
   /**
     * An array holding all the pizzas in the order

--- a/PizzaOrder.java
+++ b/PizzaOrder.java
@@ -31,7 +31,7 @@ public class PizzaOrder {
     */
   public PizzaOrder() {
     m_order = new Pizza [1];
-    m_order[0] = new Pizza("small", 1, 0, 0);
+    m_order[0] = new Pizza("small", 1, 0, 0, 0);
     m_numPizzas = 1;
   }
 

